content/blog/2021-08-31-aug-community-gems.md

```diff
@@ -0,0 +1,117 @@
+  clear logical groupings between stages. It avoids confusion, improves
+  readability, and shortens commands by avoiding long paths preceding every
+  filename.
+
+If you want to see the rest of the explanation,
+[check out this PR we have up](https://github.com/iterative/dvc.org/issues/2494).
+Please feel free to add a comment or emoji on this PR so we know how to
+prioritize this content for you!
+
+Thanks @Tups!
+
+### [Q: Is there way to allow different pipelines to have common dependencies and outputs in DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/867747202306146335)
+
+Good question @vgodie!
+
+It is possible to have overlapping dependencies, but not overlapping outputs.
+Having overlapping outputs introduces uncertainty into DVC commands, like
+`dvc checkout`.
+
+### [Q: How does the CML runner functionality to restart workflows if it's been shut down by AWS (e.g. spot instances) work?](https://discord.com/channels/485586884165107732/728693131557732403/862641924200857660)
+
+You shouldn't have to do anything. Spot instances sends a `SIGINT` that we
+handle to restart the workflow. We have been supporting graceful shutdown by
+unregistering runers for a while now.
+
+The main difference now is that we restart workflows with unfinished jobs.
+
+We know for sure this isn't working in Azure. Azure is not doing a graceful
+shutdown, but a shutdown now, even when stopping the instance within the Azure
+console.
+
+Thanks for such a good question @andee96!
+
+### [Q: Can I change the endpoint defined here <https://github.com/iterative/cml/blob/master/src/utils.js#L56>? Or does `cml-publish` always save the artifacts on this endpoint?](https://discord.com/channels/485586884165107732/728693131557732403/864444303169421322)
+
+Good question @Nwp8nice!
+
+If you use GitLab you can use the `--native` option to upload to GitLab instead.
+
+It would be nice to be able to offer an alternative link so if you're
+interested, a PR would be awesome! :blush:
+
+### [Q: Is CML used for creating the MLOps workflows, like Apache Airflow?](https://discord.com/channels/485586884165107732/728693131557732403/866624571519664128)
+
+This is a really good question @Ravi Kumar!
+
+CML relies on CI/CD engines like GitHub Actions or GitLab CI/CD, but doesn't
+replace them. It's just a lightweight wrapper and not a complete workflow
+ecosystem like Airflow.
+
+### [Q: Does CML have the ability to run long-run instances, e.g. can I run AWS instance via GitHub Actions that works more than 6 hours?](https://discord.com/channels/485586884165107732/728693131557732403/866730530262351873)
+
+You can definitely run it for 6 hours. The GitHub limit is 72 hours for
+self-hosted runners and GitLab has no limit on self-hosted runners.
+
+Thanks @sergechuvakin!
+
+---
+
+https://media.giphy.com/media/l0IycQmt79g9XzOWQ/giphy.gif
+
+At our September Office Hours Meetup we will be do something.
```

Comment:
```suggestion
At our September Office Hours Meetup we will be doing something.
```

Either reads as v. mysterious or v. obvious :laughing:

Reply:
😆 Guessing you are still trying to figure this one out @flippedcoder and @jendefig?

Reply:
Yeah, we were still working on that. 😅 But I've got that updated now!